src/Views/reportview.html

```diff
@@ -285,6 +285,38 @@ <h3 class="reportChartHeader">Estimated Flows</h3>
                         </div><!--end sc-->
                     </div><!--end extension-->
 
+                <div class="well well-sm"  ng-if="vm.regionService.selectedRegion.Applications.indexOf('EquationWeighting') != -1 && vm.nssService.selectedStatisticsGroupList.name.indexOf('Peak-Flow Statistics') != -1">
+                    <div class="wim-alert-warning" ng-show="vm.nssService.equationWeightingDisclaimers == true">Only 3 estimation methods can be weighted; the 3 estimation methods with lowest SEP values were weighted. </div>
+                    <h3>Methods Weighting</h3>
+                    <div ng-if="vm.nssService.equationWeightingResults.length == 0">No method weighting results returned</div>
+                    <div ng-if="vm.nssService.equationWeightingResults.length != 0">

```

Comment:
Before this section, include definitions (similar to other sections of the report):

"PIl: Prediction Interval-Lower, PIu: Prediction Interval-Upper, ASEp: Average Standard Error of Prediction"